docs/tutorials/integrations/web_search.md

```diff
@@ -251,43 +308,58 @@ services:
         max-file: "1"
 ```
 
-Launch your updated stack with:
+Your stack is ready to be launched with:
 
 ```bash
 docker compose up -d
 ```
 
+:::note
+On the first run, you must remove `cap_drop: - ALL` from the `docker-compose.yaml` file for the `searxng` service to successfully create `/etc/searxng/uwsgi`.ini. This is necessary because the `cap_drop: - ALL` directive removes all capabilities, including those required for the creation of the `uwsgi.ini` file. After the first run, you should re-add `cap_drop: - ALL` to the `docker-compose.yaml` file for security reasons.
+:::
+
 Alternatively, you can run SearXNG directly using `docker run`:
 
 ```bash
-docker run --name searxng --env-file stack.env -v ./searxng:/etc/searxng:rw -p 1337:8080 --restart unless-stopped --cap-drop ALL --cap-add CHOWN --cap-add SETGID --cap-add SETUID --cap-add DAC_OVERRIDE --log-driver json-file --log-opt max-size=1m,max-file=1 searxng/searxng:latest
+docker run --name searxng --env-file .env -v ./searxng:/etc/searxng:rw -p 8080:8080 --restart unless-stopped --cap-drop ALL --cap-add CHOWN --cap-add SETGID --cap-add SETUID --cap-add DAC_OVERRIDE --log-driver json-file --log-opt max-size=1m,max-file=1 docker.io/searxng/searxng:latest
```

Comment:
q: is `docker.io/` needed?

Reply:
> q: is `docker.io/` needed?

It’s not needed, no. Looks like I forgot to add it in a spot too. I’ll fix it up when I can.